### PR TITLE
Auto OCP-37827

### PR DESCRIPTION
**Pull request by chao007**
@duanwei33  Please help to review.
It is auto for prod env.

https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/231705/console

### CODE REVIEW

**Comment by duanwei33** [comment]
@chao007 
Before this case, the csi driver has been installed and images is from prod registry?

**Comment by chao007** [comment]
> @chao007
> Before this case, the csi driver has been installed and images is from prod registry?

Yes. This is the installation job https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Launch%20Environment%20Flexy/128821/

File: features/storage/csi.feature [hunk 1]
```diff
@@ -31,3 +31,16 @@ Feature: CSI testing related feature
...
+  @admin
+  Scenario: CSI images checking in prod registry env
+    Given the master version >= "4.4"
+    Given I switch to cluster admin pseudo user
+    Given admin uses the "csihostpath" project
+    Given the pod named "my-csi-app" status becomes :running
+    And the expression should be true> pvc("csi-pvc").capacity(cached: false) == "2Gi"
+    When I run the :get admin command with:
+      | resource | volumesnapshot |
+    Then the output should contain "true"
```
Comment (liangxia) [inline features/storage/csi.feature:46]: Exactly the same steps as in OCP-30787

Perhaps we can merge them into scenario outline 
```
  # @author chaoyang@redhat.com
  # @case_id OCP-30787
  @admin
  Scenario Outline: CSI images checking
    Given the master version >= "4.4"
    Given I log the message> CSI images checking in <cluster> env
    Given I switch to cluster admin pseudo user
    Given admin uses the "csihostpath" project
    Given the pod named "my-csi-app" status becomes :running
    And the expression should be true> pvc("csi-pvc").capacity(cached: false) == "2Gi"
    When I run the :get admin command with:
      | resource | volumesnapshot |
    Then the output should contain "true"
    Examples:
      | cluster |
      | stage   | # @case_id OCP-30787
      | prod    | # @case_id OCP-37827
```

File: features/storage/csi.feature [hunk 2]
```diff
@@ -31,3 +31,16 @@ Feature: CSI testing related feature
     Given I switch to cluster admin pseudo user
     Given admin uses the "csihostpath" project
     Given the pod named "my-csi-app" status becomes :running
+
+  # @author chaoyang@redhat.com
+  # @case_id OCP-37827
```
Comment (liangxia) [inline features/storage/csi.feature:36]: Exactly the same steps as in OCP-30787

Perhaps we can merge them into scenario outline
```
  # @author chaoyang@redhat.com
  # @case_id OCP-30787
  @admin
  Scenario Outline: CSI images checking
    Given the master version >= "4.4"
    Given I log the message> CSI images checking in <cluster> env
    Given I switch to cluster admin pseudo user
    Given admin uses the "csihostpath" project
    Given the pod named "my-csi-app" status becomes :running
    And the expression should be true> pvc("csi-pvc").capacity(cached: false) == "2Gi"
    When I run the :get admin command with:
      | resource | volumesnapshot |
    Then the output should contain "true"
    Examples:
      | cluster |
      | stage   | # @case_id OCP-30787
      | prod    | # @case_id OCP-37827
```

Comment (chao007) [reply features/storage/csi.feature]: Thanks for your comments and the auto scripts. Have updated it. After it merged, I will update polarion

**Pull request merged**